src/rgw/rgw_gc.h

```diff
@@ -51,7 +51,8 @@ class RGWGC : public DoutPrefixProvider {
     finalize();
   }
   std::vector<bool> transitioned_objects_cache;
-  int send_chain(cls_rgw_obj_chain& chain, const std::string& tag);
+  int send_split_chain(const cls_rgw_obj_chain& chain, const std::string& tag);
+  int send_chain(const cls_rgw_obj_chain& chain, const std::string& tag);
```

Comment:
please make `send_chain()` private, so rgw doesn't accidentally call the non-splitting version

Reply:
done